### ISSUE TITLE
Incorrect Window Size - BEATs

### ISSUE THREAD

**Issue by jeremyng353** [microsoft/unilm#991]
**Describe the bug**
Model I am using (UniLM, MiniLM, LayoutLM ...): BEATs

The problem arises when using:
* [x] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

When I try to use a finetune trained model of BEATs with my own audio file, I get the following error. The code is identical to the load Fine-tuned Models section of the README in BEATs except that I load own audio file with `torchaudio` and then use its size to create the padding mask. I've ensured that the sampling rate of the audio file is 16khz. 

![image](https://user-images.githubusercontent.com/69836033/216454948-4c888ba9-0953-470d-b287-fba47fb560f6.png)

**To Reproduce**
1. Replace `audio_input_16khz` by using `torchaudio.load()` with your own ~10s audio file.
2. Replace `padding_mask` with the size of the loaded audio file.

**Expected behavior**
Labels of the audio file should be printed.

- Platform: Google Colaboratory 
- Python version: 3.8
- PyTorch version (GPU?): 1.13.1

**Comment by Sanyuan-Chen** [comment]
Hi @jeremyng353 , please make sure the "audio_input_16khz" variable is a **torch tensor** rather than the **tuple** of (tensor, sample_rate), which is the returned value of torchaudio.load() function.
